_test/test_transformation/test_ortho_proj_construction.m

```diff
@@ -222,6 +222,7 @@ function test_get_projection_from_other_aligned_data(~)
 
         end
         function test_getset_nonortho_proj_aaa_111(~)
+            skipTest('This test is dealt with in Re #892')            
```

Comment:
this work is part of #892 and it is dealing with these tests extensively. I do not think, there is any problem with disabling these tests here.

Reply:
well, I do not understand why it become problem now it if was not a problem couple of years before that

Reply:
Because as discussed:
- it affects other people's PRs and commits and has caused test failures in my #1008 PR and I subsequently don't know if I've broken something in the merge or whether it's master playing up. 
- Subsequent branches working from master may miss breaking changes because tests are disabled which may require complicated structural fixes down the line
- Because it means that master is never available in an always working state to pull from.
- It creates a bunch of stacked up work at the end of a project cleaning up all the tests rather than being able to effectively know when a release can be done
- It inhibits simple cherry picking of changes to master thereof for a complete version

Reply:
that's fair problem, but unfortunately these changes have not broken your tests. 

you would not use non-ortho projection so your tests would not be affected.

Reply:
anyway, I've decided to deal with failures in #892